github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -717,15 +717,11 @@ periodics:
             value: "kind-k8s-sriov-1.17.0"
           - name: "GIMME_GO_VERSION"
             value: "1.13.8"
-          - name: "KUBEVIRT_PROVIDER" # for cluster-down in the command below
-            value: "kind-k8s-sriov-1.17.0"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
-        - "-c"
+        - "-ce"
         - |
```

Comment:
I think we no longer need this array element to be a multiline item, it could just be `- "automation/test.sh"`, wdyt?

Reply:
It would work like that, i left it because i prefer that in case someone need to add a command (even for debug, as we need sometimes for rehearse)
he won't need to do '&&`, otherwise we can find ourself with `&&` again in the long run
wdyt?

Reply:
ok sgtm